boilerplate/registry/index.html

```diff
@@ -0,0 +1,288 @@
+                    the <a>change request</a> can be approved.</li>
+                </ol>
+              </li>
+            </ul>
+            <p>An approved <a>change request</a> is enacted by merging its
+              related pull request into the <a>registry table</a>'s version control system
+              and publishing the updated version of the <a>registry table</a>.</p>
+          </section>
+          <section id="team-change-request-assessment">
+            <h5>Custodian is the W3C Team</h5>
+            <p>If the <a>custodian</a> is the <a>W3C Team</a>,
+              the Team MUST seek [=wide review=] of the <a>change request</a> and
+              offer a review period of at least 4 weeks,
+              before assessing from the responses received
+              if there is consensus amongst the respondents.</p>
+            <p>The Team MAY require a pull request on the <a>registry table</a>'s
+              version control system to be
+              opened as the basis of the review.
+            </p>
+            <p>If there is such consensus,
+              the Team MUST make the proposed changes.</p>
+          </section>
+        </section>
+      </section>
+      <h3>Registry Table(s)</h3>
+      <p>This section defines constraints on <a>registry tables</a>.
+        Each <a>registry table</a> consists of a set of
+        <dfn data-cite="w3c-process#registry-entry" data-lt="registry entry|registry entries">registry entries</dfn>.</p>
+      <p>The <a>registry table</a> MUST provide a link to the version control system against
+        which any issues and pull requests can be opened.</p>
+      <section id="registry-entries">
+        <h4>Registry Entries</h4>
+        <p>Each <a>registry entry</a> has a <a>status</a>, a unique <dfn>key</dfn>, and
+          if appropriate, other <dfn>fields</dfn>, for example any notes,
+          a description, or a reference to some other
+          defining entity.
+        </p>
+        <p>The <a>registry table</a> definition MUST define
+          the <a>fields</a> and the <a>key</a> to be used
+          in each <a>registry entry</a>.</p>
+
+        <section id="registry-entry-status">
+          <h5>Status</h5>
+          <p>The <a>registry entry</a> <dfn>status</dfn> field reflects the maturity of that entry.
+            Permitted values are:</p>
+          <pre>
+            Provisional
+            Final
+            Deprecated
+          </pre>
+          <p>No other values are permitted.</p>
+          <section id="provisional-entry">
+            <h6>Provisional</h6>
+            <p><a>Registry entries</a> with a <a>status</a> of <dfn><code>Provisional</code></dfn> MAY be changed or deleted.
+              Their <a>status</a> may be changed to <a><code>Final</code></a> or <a><code>Deprecated</code></a>.</p>
+            <p><a>Registry entry</a> <a>keys</a> in <a><code>Provisional</code></a> entries
+              that were later deleted MAY be reused.</p>
```

Comment:
Should keys that were previously Deprecated are removed from this operation?

Reply:
I'm not sure I understand the question, but is it relevant that there's no path from Deprecated to Provisional?

Reply:
Ah, should be related to https://github.com/w3c/ttwg/commit/05ef9ab16acc346a87bda1adf89f060efeeb49f4.
I've wondered whether we should exclude keys which transited as Provisional -> Final -> Deprecated -> Provisional.

Reply:
I deliberately did not permit Deprecated -> Provisional so this does not arise.

Reply:
Ahh, I've misread the text. sorry..